static/js/gtag.js

```diff
@@ -1,4 +1,39 @@
 window.dataLayer = window.dataLayer || [];
-function gtag(){dataLayer.push(arguments);}
+function gtag() { dataLayer.push(arguments); }
 gtag('js', new Date());
 gtag('config', 'UA-26406144-13');
+
+// Setup listening to send Google Analytics events when any element
+// with a 'data-ga-click-event' attribute is clicked.
+window.addEventListener('DOMContentLoaded', function () {
+  function findPackageName() {
+    var pageData = document.getElementsByName('pub-page-data')[0];
+    if (pageData) {
+      var pkgData = JSON.parse(atob(pageData.content)).pkgData || {};
+      var pkg = pkgData.package;
+      if (!pkg) {
+        console.log('Failed to identify package name from pageData');
+        return null;
+      }
+      return 'package:' + pkg;
+    }
+    return null;
+  }
+  var label = findPackageName() || '-';
+
+  function sendEvent(e) {
+    gtag('event', e.currentTarget.dataset.gaClickEvent, {
```

Comment:
Please add a comment here that currentTarget is the DOM Element where the event listener was added to, and not the Element that the click happened.